.github/workflows/ingestion-deploy-dev.yaml

```diff
@@ -14,7 +14,9 @@ jobs:
     uses: mbta/workflows/.github/workflows/deploy-ecs.yml@main
     with:
       app-name: lamp-ingestion
-      environment: 'dev'
+      environment: dev
+      dockerfile-path: ./py_gtfs_rt_ingestion/
+      cluster: lamp
```

Comment:
both apps are deployed to the lamp cluster. without this it will assume the service name and cluster are the same string.